kamyaD/task-scheduling-solution-front
# Fixing errors in redux

### What does this PR do?
This PR fixes issues with Redux where login does not complete due to Redux error.

### How can this be manually tested?
- clone the repo and checkout to redux branch.
- ensure that the backend is running
- run `npm install` to install  dependencies
- run `npm start` to start the application
- navigate to `localhost:3001` on your browser and enter `phone` and `password` on the resulting interface and click login.
- check on the backend terminal to see login successful with status code of 200.
